tests/unittests/OperatorTest.cpp

```diff
@@ -6593,6 +6593,58 @@ TEST_P(OperatorStatelessTest, rowwiseQuantizedSLWSTest) {
                             /* enableRowwiseQuantization */ true);
 }
 
+/// Check the correctness of the bucketize operator.
+/// First example in PyTorch's documentation
+TEST_P(OperatorTest, Bucketize1) {
```

Comment:
the only difference as far as i can tell between those two tests are inputs/expected outputs.
Could you extract that in separate method and remove repetition.